### ISSUE TITLE
Allow send messages to user, not only to channel

### ISSUE THREAD

**Issue by fedordikarev** [course-hero/slacktee#14]
Hi!
It's possible to send message to user by using "@username" as channel parameter.
For now, it's hardcoded to add # sign to channel name.
Below my patch that will add # sign only if channel name doesn't start with it nor with @ sign. So it's possible to send messages with "-c @username" option.
```
--- a/slacktee.sh
+++ b/slacktee.sh
@@ -98,7 +98,10 @@ function send_message(){
            message_attr="\"text\": \"$escaped_message\","          
        fi
 
-        json="{\"channel\": \"#$channel\", \"username\": \"$username\", $message_attr \"icon_emoji\": \":$icon:\" $parseMode}"
+       if [[ ( "$channel" != "#"* ) && ( "$channel" != "@"* ) ]]; then
+               channel="#$channel"
+       fi
+        json="{\"channel\": \"$channel\", \"username\": \"$username\", $message_attr \"icon_emoji\": \":$icon:\" $parseMode}"
         curl -X POST --data-urlencode "payload=$json" "$webhook_url" &> /dev/null
     fi
 }
```

**Comment by Chikashi-Kato** [comment]
Thank you for the great improvement idea! I didn't know we can send direct message with @username.
I will implement the function based on your patch and publish it as soon as possible.

**Comment by Chikashi-Kato** [comment]
I just published v1.2.1 which support direct messaging using '@username' channel notation.
Thank you for letting me know your great idea!!
